src/App.js

```diff
@@ -1,24 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import { Header } from './Components/Header.js'
+import { Footer } from './Components/Footer.js'
+import { Content } from './Components/Content.js'
+
+import './styles/Main.css'
```

Comment:
Since you are importing it in App, it is best to name the styling the same as the component. This should be named `App.css` instead.